### ISSUE TITLE
[Data NoBlaBla : Mise en oeuvre de YOLOv3 en PyTorch] Jean-Francois Faudi

### ISSUE THREAD

**Issue by jeffaudi** [TlseDataScience/meetup-cfp#30]
## Le speaker
Jean-Francois Faudi, responsable Deep Learning chez Airbus Defense and Space | Connected Intelligence.

* Twitter handle: @jeffaudi

## Sujet de votre conférence
Data NoBlaBla : Mise en oeuvre de YOLOv3 en PyTorch
Cas de la détection de stockage de pétrole sur images satellite SPOT

## Description de votre conférence
Cet atelier a pour objectif de montrer concrètement comment appliquer un algorithme standard comme YOLOv3 sur des images satellites. Compte-tenu du temps limité (4 heures) comparé au temps normalement consacré à ce type de sujet, nous allons présenter les grandes lignes de chaque élément de la solution et faire des petits focus techniques. L'objectif est que vous soyez capables de relancer vous même des entrainements à la fin de l'atelier.

Contenu:
* Mise en place de l'environnement de travail
* Analyse des données d'entrainement
* Présentation des divers modules de la solutions
* Entrainements rapides pour présenter les hyper-paramètres
* Analyse des résultats d'un entrainement long
* Visualisation des résultats
* Packaging de la solution
* Discussion à propos des étapes suivantes

Cet atelier est donné deux fois. Vous pouvez choisir le vendredi après-midi OU le samedi matin.

## Informations diverses
* Thématique, Labels : pytorch, python, deep learning, cnn, YOLOv3, detection, images satellites
* Niveau de difficulté (débutant|avancé|confirmé) : intermédiaire, connaissances Python requises
* Durée (quickie 15-30min, conférence >1h) : atelier de 2 x 3 heures
* Format (slides, live-coding, les deux): atelier, notebook, live-coding
* Dispo ou indispo (après mars, dès que possible, pas entre X et Y): 17 et 18 janvier
